pages/release-process.md

```diff
@@ -144,6 +144,72 @@ When the vote passes on a release candidate, follow the steps below to complete
 
  8.  Send an email to `dev@fluo.apache.org` announcing new release.
 
+### Test a Fluo release
+
+Below are two common testing strategies to verify a Fluo release candidate:
+
+1. Run the integration tests of projects that depend on Fluo (using the release candidate jars)
+
+1. Run example Fluo applications (using the release candidate tarball)
```

Comment:
No need for both a bulleted list here, and then the sections below. It is redundant, and wordy. This bulleted list can go away completely.